package.json

```
@@ -237,6 +237,11 @@
         "title": "Reload KNX",
         "category": "Home Assistant"
       },
+      {
+        "command": "vscode-home-assistant.templateReload",
+        "title": "Reload Template Sensors",
```

Small suggestion, as this reload isn't limited to Sensor, but covers most other entity platforms supported by the Template integration as well.

```suggestion
        "title": "Reload Template Entities",
```

---

Thanks for the quick review and good feedback, I wasn't really aware of that so then I learned something today 👍